src/components/Home.js

```diff
@@ -67,6 +67,10 @@ class Home extends Component {
               <h3 css={tw`text-4xl font-light text-gray-700`}>
                 What to expect
               </h3>
+              <h4 css={tw`text-xl font-light text-gray-700`}>
```

Comment:
Semantically this isn't really a heading, so we can keep it visually the same while changing the markup:
```suggestion
              <span css={tw`block text-xl font-light text-gray-700`}>
```

Reply:
Change made